Week2Project/src/test/java/test/suites/CrocodileTests.java

```diff
@@ -41,55 +64,45 @@ public void getListOfPublicCrocodiles() {
     public void getOnePublicCrocodile() {
         int r = getRandomNumber(1, 8);
         String id = Integer.toString(r);
-        GetOnePublicCrocodileResponse getOnePublicCrocodileResponse = CrocodilesAPI.getOnePublicCrocodileResponse(id);
+        CrocodileResponse getOnePublicCrocodileResponse = CrocodilesAPI.getOnePublicCrocodileResponse(id);
         crocodileAsserts.assertOnePublicCrocodile(getOnePublicCrocodileResponse);
     }
 
     @Test
     @Description("list of my private crocodiles")
     public void getListOfPrivateCrocodiles() {
-        GetPrivateCrocodileResponse[] getPrivateCrocodileResponse = CrocodilesAPI.getPrivateCrocodileResponse(accessToken);
+        CrocodileResponse[] getPrivateCrocodileResponse = CrocodilesAPI.getPrivateCrocodileResponse(accessToken);
         crocodileAsserts.assertListOfPrivateCrocodiles(getPrivateCrocodileResponse);
     }
 
     @Test
     @Description("one of my crocodiles")
     public void getOneOfMyCrocodiles() {
-        String firstId = getFirstID();
-
-        GetOneOfMyCrocodilesResponse getOneOfMyCrocodilesResponse = CrocodilesAPI.getOneOfMyCrocodilesResponse(firstId, accessToken);
+        CrocodileResponse getOneOfMyCrocodilesResponse = CrocodilesAPI.getOneOfMyCrocodilesResponse(String.valueOf(idOfCreatedCrocodile), accessToken);
         crocodileAsserts.assertOneOfMyCrocodiles(getOneOfMyCrocodilesResponse);
     }
 
     @Test
     @Description("update one of my crocodiles")
     public void updateCrocodile() {
-        String firstId = getFirstID();
-        int id = Integer.parseInt(firstId);
-
-        UpdateCrocodileRequest updateCrocodileRequest = new UpdateCrocodileRequest(id, "Ana", "F", "2007-05-05", 16);
-        UpdateCrocodileResponse updateCrocodileResponse = CrocodilesAPI.updateNewCrocodileResponse(firstId, accessToken, updateCrocodileRequest);
+        UpdateCrocodileRequest updateCrocodileRequest = new UpdateCrocodileRequest(idOfCreatedCrocodile, Faker.instance().gameOfThrones().character(), ValueChoosers.getRandomSex(), DF.format(Faker.instance().date().birthday()), 16);
```

Comment:
I za update croc request mozes pozivati onu metodu koja to pravi random request, jer isti je request i za create i za update